### PR TITLE
Turned registration 'on'

### PR DESCRIPTION
**Pull request by r1g023**
* Turned 'on' user registration, registering a user is functional 
* Removed LandingPage - replaced with LoginContainer instead
* Styled the signup form, needs more styling so that it matches the login form. 
* Email verification is working but needs work through Okta settings on redirecting to logged in app. Email verification also needs to be designed to show it's coming from Story Squad and not Okta.

### CODE REVIEW

File: src/styles/less/LoginContainer.less [hunk 1]
```diff
@@ -15,59 +15,59 @@
...
-    font-weight: 700;
-    font-size: 1rem;
-    letter-spacing: 1.25px;
-    text-transform: uppercase;
-    display: flex;
-    align-items: center;
+    outline: none;
+    border: 1px solid #7b8794;
   }
+  // a.registration-link {
```
Comment (ianpaulfo) [inline src/styles/less/LoginContainer.less]: Wondering if the code thats commented out is the registration styling that is in progress or should this be deleted?

**Review by ianpaulfo — COMMENTED** [review]
Commits look good, short and specific.

We need a description for the pull request summarizing what was done.

left a comment on less/login container regarding code that is commented out on line 54

File: src/styles/less/LoginContainer.less (hunk 1 above)
Comment (r1g023) [reply src/styles/less/LoginContainer.less]: @ianpaulfo I have removed the commented code from LoginContainer.less now, it wasn't needed.

Comment (r1g023) [reply src/styles/less/LoginContainer.less]: @ianpaulfo @tinomen I've added a brief description to the pull request

**Comment by tinomen** [comment]
Closing PR for now, looking into alt registration.

**Pull request closed without merging**